core/cat/mad_hatter/mad_hatter.py

```diff
@@ -137,14 +143,14 @@ def embed_tools(self):
     # Tries to load the plugin metadata from the provided plugin folder
     def get_plugin_metadata(self, plugin_folder: str):
         plugin_id = path.basename(plugin_folder)
-        plugin_json_metadata_file_name = "plugin.json"
-        plugin_json_metadata_file_path = path.join(plugin_folder, plugin_json_metadata_file_name)
-        meta = {"id": plugin_id}
+        metadata_file_name = "plugin.json"
+        metadata_file_path = path.join(plugin_folder, metadata_file_name)
+        meta = { "id": plugin_id }
```

Comment:
@zAlweNy26 tuc tuc tuc, not pythonic...🐍

Reply:
pyth...what? @nicola-corbellini 🤒